english/formosa/taiwan-story.md

```diff
@@ -0,0 +1 @@
+Did you know that Taiwan has its own "Anne Frank" story about a man in Hsinchu who hid in a small, secret hiding place -- a thin space between two walls, with no room to even stand up -- for 18 years during the White Terror period?
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.